### PR TITLE
feat(client): board support show project tree and model list

### PR DESCRIPTION
**Pull request by jialeicui**
## Description

https://user-images.githubusercontent.com/3217223/178645613-ed8a2983-ce1a-4f4a-87f0-abe825b3ee56.mp4

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/cli/board/board.py [hunk 1]
```diff
@@ -122,7 +121,8 @@ async def key_h(self):
 class Models(TableWidget):
     """Models represents starwhale model view"""
 
-    def __init__(self, **kwargs) -> None:
+    # TODO use constance
```
Comment (tianweidut) [inline client/starwhale/cli/board/board.py:124]: `constance` typo?  instance or other words？

**Review by tianweidut — COMMENTED** [review]
Others LGTM

File: client/starwhale/cli/board/board.py (hunk 1 above)
Comment (jialeicui) [reply client/starwhale/cli/board/board.py]: constant :smiling_face_with_tear: 

changed to `f"{DEFAULT_INSTANCE}/{DEFAULT_PROJECT}"`

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**